### PR TITLE
tests(cleanup): enabling some tests, removing some tests

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Enhancement**

This PR resolves []

### Description

Removing some tests that have been incomplete for a long time, or skipped for a long time.

Also enabling a few tests that were skipped.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

### Changelog updates

#nochangelog
#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5b2fee455e1467f6af39ad31ef2e3c48492121ec
</p>

**Review by gkartalis — APPROVED** [review]
approved on the JS side, ios tests though fail for some reason 🤔 more specifically they have this error on the checks:

`expected a matching snapshot in updated_message
Artsy_Tests/View_Controller_Tests/SaleOnHoldOverlayViewTests.swift:72`

**Review by MounirDhahri — DISMISSED** [review]

**Review by MounirDhahri — APPROVED** [review]

File: Artsy_Tests/View_Controller_Tests/SaleOnHoldOverlayViewTests.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ class SaleOnHoldOverlayViewTests: QuickSpec {
                 subject.align(toView: containerView!)
             }
 
-            xit("handles subsequent message updates") {
+            it("handles subsequent message updates") {
```
Comment (pvinis) [inline Artsy_Tests/View_Controller_Tests/SaleOnHoldOverlayViewTests.swift:65]:
```suggestion
            xit("handles subsequent message updates") {
```
not worth fighting for.

**Review by MounirDhahri — APPROVED** [review]
Thanks for enabling this again!

**Review by brainbicycle — APPROVED** [review]
Thanks!

**Pull request merged**